src/web/layouts/StandardLayout.tsx

```diff
@@ -548,37 +551,56 @@ export const StandardLayout = ({
                         </ArticleContainer>
                     </GridItem>
                     <GridItem area="right-column">
-                        <RightColumn>
-                            <StickyAd
-                                name="right"
-                                height={MOSTVIEWED_STICKY_HEIGHT}
-                            />
-                            {!isPaidContent ? <MostViewedRightIsland /> : <></>}
-                        </RightColumn>
+                        <div
+                            className={css`
+                                padding-top: 6px;
+                            `}
+                        >
+                            <RightColumn>
+                                <StickyAd
+                                    name="right"
+                                    height={MOSTVIEWED_STICKY_HEIGHT}
+                                />
+                                {!isPaidContent ? (
+                                    <MostViewedRightIsland />
+                                ) : (
+                                    <></>
+                                )}
+                            </RightColumn>
+                        </div>
                     </GridItem>
                 </StandardGrid>
             </Section>
 
             <Section
-                data-print-layout='hide'
+                data-print-layout="hide"
                 padded={false}
                 showTopBorder={false}
                 showSideBorders={false}
                 backgroundColour={neutral[93]}
             >
-                <AdSlot data-print-layout='hide' asps={namedAdSlotParameters('merchandising-high')} />
+                <AdSlot
+                    data-print-layout="hide"
+                    asps={namedAdSlotParameters('merchandising-high')}
+                />
             </Section>
 
             {!isPaidContent && (
                 <>
```

Comment:
unrelated but maybe worth moving ` data-print-layout="hide"` here?

Reply:
That change popped up here because of Prettier glitches elsewhere. I'm only vaguely aware what it's about tbh!